consensus/hotstuff/verification/combined_verifier.go

```diff
@@ -128,9 +107,17 @@ func (c *CombinedVerifier) VerifyQC(voterIDs []flow.Identifier, sigData []byte,
 		return false, nil
 	}
 	// verify the aggregated staking signature next (more costly)
-	// TODO: eventually VerifyMany will be repalced by Verify from Verifier
-	// TODO: the agg public key would be computed outside verify() based on the delta
-	stakingValid, err := c.staking.VerifyMany(msg, stakingAggSig, signers.StakingKeys())
+	// TODO: eventually VerifyMany will be a method of a stateful struct. The struct would
+	// hold the message, all the participants keys, the latest verification aggregated public key,
+	// as well as the latest list of signers (preferably a bit vector, using indices).
+	// VerifyMany would only take the signature and the new list of signers (a bit vector preferably)
+	// as inputs. A new struct needs to be used for each epoch since the list of participants is upadted.
```

Comment:
I've left this comment as a hint to the upcoming stateful API refactor. I'm not sure yet where the stateful struct for staking signatures will live, I'll try to mirror the APIs of the beacon threshold signature. This will be the topic of a future ticket.